jest.config.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
-  transform: { "^.+\\.(t|j)sx?$": "@swc/jest" },
+  transform: { "^.+\\.(t|j)sx?$": "ts-jest" },
```

Comment:
Opted to use `ts-jest` as I could not `spyOn` the validate module (similar to; https://github.com/swc-project/swc/issues/5059), which will need to be resolved by the transformation function on client side (see: https://github.com/swc-project/swc/discussions/5151)